hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTimeline.java

```diff
@@ -42,6 +42,10 @@
   String ROLLBACK_ACTION = "rollback";
   String SAVEPOINT_ACTION = "savepoint";
   String INFLIGHT_EXTENSION = ".inflight";
+  // With Async Compaction, compaction instant can be in 3 states :
+  // (compaction.requested), (compaction.inflight), (.commit)
```

Comment:
Can we avoid using "commit" as a state ? This may cause confusion.

Reply:
+1. lets use `completed` instead

Reply:
Done